contracts/Interfaces/UniswapInterfaces/IUniSimple.sol

```diff
@@ -1,5 +1,9 @@
+// SPDX-License-Identifier: UNLICENSED
 pragma solidity >=0.6.2;
```

Comment:
I would replace all `pragma solidity >= something;` with `pragma solidity 0.6.12;` so we have a single version for everything.